source/install/self-managed-changelog.md

```diff
@@ -5,15 +5,154 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+
+#### Emoji Enhancements with Skin Tone Selection
+ - Added support for emoji standard v13.0. Users now have the ability to choose various skin tones using the Mattermost emoji picker. Mobile support is included in v1.45 Mobile App release (July 16th).
+
+#### Improved Enterprise Trial Experience (Enterprise Editions E0, E10, E20)
+ - After a Self-Managed trial ends, admins can optionally contact sales or make a purchase in a single click.
+
+#### Focalboard: Grouped Table view, New properties, and More (Beta)
+ - Focalboard tables can now be grouped by a property, for example allowing you to quickly see tasks per epic or owner.
+
+#### Incident Collaboration Updates
+ - The update includes availability on all editions, playbook keyword monitoring, retrospective report, and playbook dashboard.
+
+#### English-Australian Language Support
+ - Mattermost is now available in English-Australian.
+
+### Improvements
+
+#### User Interface (UI)
+ - In the at-mention autocomplete, the user’s nickname is no longer shown when (you) is present.
+ - Updated the help text on the **Add Users** channel modal.
+ - Added the ability to upload ``.jpeg`` files on Linux. Uploading ``.jpg`` files was already supported.
+ - The **Channel Switcher** now displays recently viewed channels when launched.
+ - Polish, German, and Italian languages were downgraded to beta as they are [no longer actively maintained](https://handbook.mattermost.com/contributors/contributors/localization#translation-quality).
+ - Custom statuses can now be set to expire after common time intervals or custom selected dates and times. Mobile App support will be added in a future release.
+
+#### Performance
+ - Added some improvements to typing performance.
+
+#### Administration
+ - Improved memory performance for large image uploads, particularly PNGs with transparency.
+ - Optimized the bulk import process by no longer requiring the server to write the incoming archive to the filesystem when unzipping it.
+ - Added channel restore and channel privacy change endpoints to the local mode using the System bot.
+
+### Bug Fixes
+ - Fixed an issue where users were unable to set a custom status emoji via slash command. Added the logic for detecting unicode emoji and setting it as a custom status emoji via slash commands.
+ - Fixed an issue where messages with fallback text were repeated.
+ - Fixed an issue where a persistent unread badge showed on the **Main Menu** when **Enable Marketplace** or **Enable Plugins** was disabled.
+ - Fixed an issue where sidebar icons were not aligned with the navigator area icons.
+ - Fixed an issue where using CTRL+F in a **Direct Message** channel added the user ID rather than the user's name into the search field.
+ - Fixed an issue where user icons were displayed at full opacity in muted channels.
+ - Fixed an issue where a redundant ``user_update`` websocket event was generated for bot users.
+
```

Comment:
Can we add this to the changelog -- I can submit a separate PR as needed

```
- Fixed an issue where a Self-Managed Enterprise Edition E20 trial could be activated more than once.
```

@amyblais I know I originally removed it from improvements, but I'm thinking we can list it as a bug fix to avoid surprises, which is not the intent.